packages/glibc_headers.rb

```diff
@@ -3,7 +3,7 @@
 class Glibc_headers < Package
   description 'GNU C Library Headers'
   homepage 'https://www.gnu.org/software/libc/'
-  version '2.27'
+  version '2.27-0'
```

Comment:
No need to bump the version since this hasn't been merged yet.

Reply:
True.